package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "universal-data-tool",
-  "version": "0.14.10",
```

Comment:
these shouldnt be manually changed, any reason why you changed it? maybe something was breaking?

Reply:
Probably changed during the merge so i will just changed it back